google/cloud/bigtable/examples/bigtable_samples_instance_admin.cc

```diff
@@ -136,6 +136,26 @@ void DeleteInstance(google::cloud::bigtable::InstanceAdmin instance_admin,
 }
 //! [delete instance]
 
+//! [create cluster]
+// Before creating cluster, need to create instance first, then create cluster
+// on it.
+void CreateCluster(google::cloud::bigtable::InstanceAdmin instance_admin,
```

Comment:
Actually, can you change the script in this directory to actually call this example?